examples/simple-app/index.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import ReactDOM from 'react-dom'
+import { Heading, Layout } from 'virtuoso-design-system/dist/bundle.esm'
+import 'virtuoso-design-system/dist/styles.css'
```

Comment:
We should consider adding these JS and CSS imports to the top-level README… (or even in Storybook [intro for engineers](https://avkvirtru.github.io/virtuoso-design-system/?path=/docs/basics-for-engineers-🧰--page))

Reply:
I agree!